python/tk_desktop/desktop_window.py

```diff
@@ -158,6 +158,16 @@ def __init__(self, parent=None):
         url_action = self.user_menu.addAction(connection.base_url.split("://")[1])
         self.user_menu.addSeparator()
         advanced_menu = self.user_menu.addMenu("Advanced")
+
+        # We will end up with a disable advanced menu item for the
+        # debug logging toggle if TK_DEBUG is set in the environment,
+        # so we want it to be a bit more obvious that the action is
+        # disabled than we get out of the box.
+        advanced_menu.setStyleSheet(
```

Comment:
This like we're putting the advanced_menu darker instead of the toggle. Is that on purpose? There are other things in that menu that is always availble like "Show Console..."

Reply:
could this be in the main stylesheet file?